docs/_pages/quick-start-guide.md

```diff
@@ -75,7 +75,9 @@ module.exports = {
   testPath: 'tests',
   components: 'components',
   outputPath: 'tests',
-  technique: 'actionFocused',
+  planner: {
```

Comment:
Looks like you keep the config flat, but keep a planning object here. Also can you update components to componentPath (i missed it when i made the change)